art/attacks/evasion/boundary.py

```diff
@@ -300,8 +300,7 @@ def _orthogonal_perturb(self, delta, current_sample, original_sample):
         :rtype: `np.ndarray`
         """
         # Generate perturbation randomly
-        # input_shape = current_sample.shape
-        perturb = np.random.randn(*self.classifier.input_shape).astype(ART_NUMPY_DTYPE)
+        perturb = np.random.randn(*current_sample.shape).astype(ART_NUMPY_DTYPE)
```

Comment:
Why did you change from `classifier.input_shape` to `current_sample_shape`? Shouldn't they be the same?

Reply:
Yes, they are basically the same. But as we talked before, when the Keras classifier is not fixed with a prediction that doesn't check shape, inputshape maybe different than sample shape. And then it caused an error. So now, that problem was fixed so then this change is not needed anymore. Do you want to go back with the previous one or still keep this as they are not different.